src/multidim/intervalbox.jl

```diff
@@ -134,3 +134,12 @@ as a vector.
         nodes
     end
 end
+
+
+function IntervalArithmetic.hull(a::IntervalBox{N,T}, b::IntervalBox{N,T}) where {N,T}
+   v = Interval{T}[]
+   for ind in 1:N
+       push!(v, hull(a[ind], b[ind]))
```

Comment:
You don't want to be using an array here, since that will allocate and be inefficient.
Instead just use broadcasting, which is specialised for `SVector`s (on which `IntervalBox`es are based). 

Also note that this is already implemented as `union` of two `IntervalBox`es.

In fact, probably `hull` should just be an alias for `union`?

Reply:
Good catch, thanks!. Done in the last commit.